### PR TITLE
Updated developers list

### PR DESCRIPTION
**Pull request by jonbullock**

### CODE REVIEW

File: gradle/maven-publishing.gradle [hunk 1]
```diff
@@ -20,7 +20,18 @@ def jbakePom = pom {
                 name 'Jonathan Bullock'
                 email 'jonbullock@gmail.com'
                 url 'http://jonathanbullock.com'
-                timezone '0'
+                timezone 'Europe/London'
+            }
+            developer {
+                id 'ancho'
+                name 'Frank Becker'
```
Comment (ancho) [inline gradle/maven-publishing.gradle]: you can add my email address frank@calmdevelopment.de, thanks.

**Comment by jonbullock** [comment]
Hmm getting `java.lang.OutOfMemoryError: Direct buffer memory` in the build:

```
org.jbake.app.template.GroovyTemplateEngineRenderingTest > renderPost STANDARD_OUT
    13:14:16.775 ERROR c.o.o.c.s.i.m.ODirectMemoryStorage - $ANSI{green {db=documents1520860456078}} Exception 35736796 in storage memory:documents1520860456078: 2.2.30 (build 43d850df23c0fd1edca7279761c2238fdc6cc576, branch 2.2.x)
    java.lang.OutOfMemoryError: Direct buffer memory
    	at java.base/java.nio.Bits.reserveMemory(Bits.java:187)
    	at java.base/java.nio.DirectByteBuffer.<init>(DirectByteBuffer.java:123)
    	at java.base/java.nio.ByteBuffer.allocateDirect(ByteBuffer.java:310)
    	at com.orientechnologies.common.directmemory.OByteBufferPool.allocateBuffer(OByteBufferPool.java:410)
    	at com.orientechnologies.common.directmemory.OByteBufferPool.acquireDirect(OByteBufferPool.java:361)
    	at com.orientechnologies.orient.core.storage.impl.memory.ODirectMemoryOnlyDiskCache$MemoryFile.addNewPage(ODirectMemoryOnlyDiskCache.java:495)
    	at com.orientechnologies.orient.core.storage.impl.memory.ODirectMemoryOnlyDiskCache$MemoryFile.access$200(ODirectMemoryOnlyDiskCache.java:458)
    	at com.orientechnologies.orient.core.storage.impl.memory.ODirectMemoryOnlyDiskCache.allocateNewPage(ODirectMemoryOnlyDiskCache.java:247)
    	at com.orientechnologies.orient.core.storage.impl.local.paginated.atomicoperations.OAtomicOperation.commitChanges(OAtomicOperation.java:434)
    	at com.orientechnologies.orient.core.storage.impl.local.paginated.atomicoperations.OAtomicOperationsManager.endAtomicOperation(OAtomicOperationsManager.java:468)
    	at com.orientechnologies.orient.core.storage.impl.local.paginated.atomicoperations.OAtomicOperationsManager.endAtomicOperation(OAtomicOperationsManager.java:412)
    	at com.orientechnologies.orient.core.storage.impl.local.paginated.base.ODurableComponent.endAtomicOperation(ODurableComponent.java:116)
    	at com.orientechnologies.orient.core.storage.impl.local.paginated.OPaginatedCluster.create(OPaginatedCluster.java:193)
    	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.addClusterInternal(OAbstractPaginatedStorage.java:4349)
    	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.doAddCluster(OAbstractPaginatedStorage.java:4330)
    	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.create(OAbstractPaginatedStorage.java:488)
    	at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.create(ODatabaseDocumentTx.java:439)
    	at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.create(ODatabaseDocumentTx.java:399)
    	at com.orientechnologies.orient.core.db.OPartitionedDatabasePool.openDatabase(OPartitionedDatabasePool.java:305)
    	at com.orientechnologies.orient.core.db.OPartitionedDatabasePool.acquire(OPartitionedDatabasePool.java:269)
    	at org.jbake.app.ContentStore.<init>(ContentStore.java:59)
    	at org.jbake.app.DBUtil.createDataStore(DBUtil.java:15)
    	at org.jbake.app.template.AbstractTemplateEngineRenderingTest.setup(AbstractTemplateEngineRenderingTest.java:101)
    	at jdk.internal.reflect.GeneratedMethodAccessor39.invoke(Unknown Source)
    	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
    	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
    	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
    	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
    	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
    	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
    	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
    	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
    	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
    	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
    	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
    	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
    	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
    	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
    	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
    	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
    	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
    	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
    	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
    	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
    	at jdk.internal.reflect.GeneratedMethodAccessor27.invoke(Unknown Source)
    	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
    	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
    	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
    	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
    	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
    	at com.sun.proxy.$Proxy3.processTestClass(Unknown Source)
    	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
    	at jdk.internal.reflect.GeneratedMethodAccessor26.invoke(Unknown Source)
    	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
    	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
    	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
    	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
    	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
    	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
    	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
    	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
    	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
    	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
    	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
    	at java.base/java.lang.Thread.run(Thread.java:844)

org.jbake.app.template.GroovyTemplateEngineRenderingTest > renderPost FAILED
    java.lang.OutOfMemoryError: Direct buffer memory
    java.lang.NullPointerException
```

**Comment by ancho** [comment]
Hmm. Are you using jdk9?

Try adding `org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:MaxDirectMemorySize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8`
To your gradle.properties file and play around with MaxDirectMemorySize.

But I think we have a deeper problem here. Which maybe a problem of gradle's caching solution.
Alternatively try to stop the gradle daemon first ./gradlew --stop

Do you mind opening an issue to track the problem?

**Comment by manikmagar** [comment]
There are some other failure causes too. It may be better to investigate and track JDK9 build failure in a separate issue.  For this particular PR, I think it is ok to merge.

```
org.jbake.app.template.GroovyTemplateEngineRenderingTest > renderArchive FAILED
    com.orientechnologies.orient.core.exception.ODatabaseException: Cannot create database 'documents1520861836425'

        Caused by:
        com.orientechnologies.orient.core.index.OIndexException: Cannot create the index 'dictionary'
        	DB name="documents1520861836425"

            Caused by:
            com.orientechnologies.orient.core.exception.OStorageException: File with name dictionary.sbt already exists.
            	DB name="documents1520861836425"

    java.lang.NullPointerException

```

**Pull request merged**